guides/common/modules/con_content-view-environment-ordering-and-priority.adoc

```diff
@@ -0,0 +1,65 @@
+[id="content-view-environment-ordering-and-priority"]
+= Content view environment ordering and priority
+
+When you assign multiple content view environments to a host or activation key, the order you specify determines the priority of the content.
+An activation key with multiple content view environments assigns them to hosts during registration in the same order as they are stored on the activation key.
+
+.Importance of content view environment order
+The order of assigned content view environments is critical in scenarios involving repository conflicts.
+A conflict occurs when two or more content view environments contain a repository with the same repository label.
+For example:
+A host assigned to the content view environments `Library/cv1, dev/cv1` uses the following configuration:
+[options="nowrap" subs="+quotes"]
+----
+hammer host update \
+--content-view-environments "Library/cv1, dev/cv1" \
+--name "{server-example-com}" \
+--organization-id _My_Organization_ID_
+----
+Both content view environments include the `satellite-client-6-for-rhel-9-x86_64-rpms` repository.
+Log in as a `root` user and when you inspect the host's available repositories using the `subscription-manager repos` command, the output reflects the `Library/cv1` content view environment:
+[options="nowrap" subs="+quotes"]
+----
+# subscription-manager repos
++----------------------------------------------------------+
+    Available Repositories in /etc/yum.repos.d/redhat.repo
++----------------------------------------------------------+
+Repo ID:   satellite-client-6-for-rhel-9-x86_64-rpms
+Repo Name: Red Hat Satellite Client 6 for RHEL 9 x86_64 (RPMs)
+Repo URL:  https://centos9-katello-devel-stable.example.com/pulp/content/Default_Organization/Library/cv1/content/dist/layered/rhel9/x86_64/sat-client/6/os
+Enabled:   0
+----
+
+[NOTE]
+====
+The `Repo URL` reflects the `Library/cv1` content view environment, meaning the system uses content from `Library/cv1` and ignores `dev/cv1`.
+====
+
+.Reordering content view environments
+Reorder the content view environments to change their priority:
+
+[options="nowrap" subs="+quotes"]
+----
+hammer host update \
+--content-view-environments "Library/cv1, dev/cv1" \
+--name "{server-example-com}" \
+--organization-id _My_Organization_ID_
+----
+After reordering, inspecting the repositories again shows the `dev/cv1` content view environment:
```

Comment:
Think this may be a bad copy/paste. We're reordering, so the command needs to be ordered differently:
```suggestion
Reorder the content view environments to change their priority:

[options="nowrap" subs="+quotes"]
----
hammer host update \
--content-view-environments "dev/cv1,Library/cv1" \
--name "{server-example-com}" \
--organization-id _My_Organization_ID_
----
After reordering, inspecting the repositories again shows the `dev/cv1` content view environment:
```